MADRaS/envs/gym_madras.py

```diff
@@ -31,6 +31,7 @@
 import MADRaS.utils.reward_manager as rm
 import MADRaS.utils.done_manager as dm
 import MADRaS.utils.observation_manager as om
+import MADRaS.envs.torcs_server_config as torcs_config
```

Comment:
Can we move the `torcs_server_config.py` into `utils`. 
This is the same old `gym` issue

Reply:
Done